lib/engine/g_1856/corporation.rb

```diff
@@ -45,11 +47,8 @@ def capitalization_type_desc
         CAPITALIZATION_STRS[@capitalization || _capitalization_type]
       end
 
-      def release_escrow!
-        @log << "Releasing #{@game.format_currency(@escrow)} from escrow for ${@name}"
-        @cash += @escrow
-        @escrow = nil
-        @capitalization = :incremental
+      def total_shares
```

Comment:
this doesn't seem like it needs to be in corporation class

Reply:
This is overriding `total_shares` from the base Corporation class

Reply:
what was wrong with the initial implementation?

Reply:
It helps with some of the weird edge cases in 1856 but since I cut off this PR before I started to tackle them I'm going to remove that from here